google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/sdk/CloudSdkPanel.java

```diff
@@ -45,6 +54,11 @@
   private JTextPane warningMessage;
   private JPanel cloudSdkPanel;
   private JLabel warningIcon;
+  private JRadioButton managedRadioButton;
```

Comment:
what determines if "managed" or "custom" will be selected by default? is this for a different PR?

Reply:
Yes, for the next PR - I'll add settings that will check and select proper value.